src/components/related-posts/related-posts-card.module.scss

```
@@ -0,0 +1,33 @@
+@import "src/tokens/index";
+
+.container {
+	display: flex;
+	padding: var(--spc-4x);
+	flex-direction: column;
+	align-items: flex-start;
+	gap: var(--spc-1x);
+	border-radius: var(--corner-radius_m);
+	text-decoration: none;
+}
+
+.container:hover {
+	background-color: var(--surface_primary_emphasis-low);
+}
+
+.container:active {
+	background-color: var(--surface_primary_emphasis-low);
+	outline: var(--spc-1x) solid var(--surface_primary_emphasis-high);
+}
+
+.container:focus-visible {
+	outline: var(--spc-1x) solid var(--focus-outline_primary);
+	background: var(--light-background-focus, #FFF);
```

```suggestion
	background: var(--background_focus);
```

Huh, I didn't know `var()` allowed default variables. I'm pretty sure this should be `--background_focus` though (which is themed), since `--light-background-focus` isn't declared anywhere?